src/browser/api/browser_view.ts

```diff
@@ -108,6 +133,13 @@ export async function setBounds(ofView: OfView, bounds: Rectangle) {
 export function getInfo (ofView: OfView) {
     return getWebContentsInfo(ofView.view.webContents);
 }
+
 export function getCurrentWindow(ofView: OfView) {
     return ofView.target;
+}
+
+export function addEventListener({uuid, name}: Identity, type: string, listener: (...args: any) => void) {
+    const eventString = route.view(type, uuid, name);
+    of_events.on(eventString, listener);
```

Comment:
Do we need to use safeListener like other entity types?

Reply:
Great point, added.